open_humans/templates/account/signup.html

```diff
@@ -1,30 +1,33 @@
 {% extends 'panel.html' %}
 
+{% load utilities %}
+
 {% block head_title %}Create an Open Humans account{% endblock %}
 
 {% block panel_content %}
-<div class="row">
-  <div class="col-xs-12">
-    {% include 'account/signup-form.html' %}
-  </div>
-</div>
 
-<div class="row">
-  <div class="col-xs-12 text-right">
-    <input type="submit" id="create-account" form="signup-form"
-      class="btn btn-primary" value="Create my account">
-  </div>
+<div class="col-md-auto">
```

Comment:
This change removed a parent row div. Column divs should be inside row divs, per https://getbootstrap.com/docs/3.3/css/